doc/source/tutorials/instrument.rst

```diff
@@ -89,12 +89,12 @@ Let's see a minimal example:
 
     from proprietary_instruments import ControllerType, controller_driver
 
-    from qibolab.components import Config
-    from qibolab.execution_parameters import ExecutionParameters
-    from qibolab.identifier import Result
-    from qibolab.sequence import PulseSequence
-    from qibolab.sweeper import ParallelSweepers
-    from qibolab.instruments.abstract import Controller
+    from qibolab._core.components import Config
+    from qibolab._core.execution_parameters import ExecutionParameters
+    from qibolab._core.identifier import Result
+    from qibolab._core.sequence import PulseSequence
+    from qibolab._core.sweeper import ParallelSweepers
+    from qibolab._core.instruments.abstract import Controller
```

Comment:
Maybe we will need a different strategy for this as well, to avoid the usage of private elements.

An option could be even to retain this as it is, because we're allowing drivers only inside Qibolab (at least for the time being), but move it to a different section, because it is then info for the Qibolab developer, rather than for the Qibolab user.

Reply:
Now covered by #1055